### DISCUSSION TITLE
installation error

### DISCUSSION THREAD

**Discussion by lsnnt** [termux/termux-packages#17612] [Q&A] [answered]
adb: failed to install com.termux.api_51.apk: Failure [INSTALL_FAILED_SHARED_USER_INCOMPATIBLE: Reconciliation failed...: Reconcile failed: Package com.termux.api has no signatures that match those in shared user com.termux; ignoring!]

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#Installation
